### PR TITLE
feat: Typography

### PR DESCRIPTION
**Pull request by dikshadhote**
Following features are added:
1.HTML tag headings
2. classes for headings
3.
image

3. font weight classes
image

preview:
https://dreamui-typography.netlify.app/documents/documents.html#typography

### CODE REVIEW

File: utility.css [hunk 1]
```diff
@@ -313,6 +313,12 @@ small {
...
 }
 
+.font-weight-regular{
+ font-weight: 400;
+}
+.font-weight-light{
+  font-weight: 300;
+}
 .code-snippet {
   background-color: #151718;
```
Comment (kaustubh-jsr) [inline utility.css]: You can use variable here, if this seems reusable.

**Review by kaustubh-jsr — COMMENTED** [review]
The UI is really neat. Code looks good to merge too.

File: utility.css (hunk 1 above)
Comment (dikshadhote) [reply utility.css]: Its only used at one place and it is not reusable, still I have changed it to color variable now

File: documents/documents.html [hunk 2]
```diff
@@ -137,8 +138,66 @@ <h2>Installation Guide</h2>
...
+            <h5>h5 heading dreamUI</h5>
+            <h6>h6 heading dreamUI</h6>
+          </div>
+          <div class="code-snippet">
+            <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%2821%2C23%2C24%2C1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=100px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ch1%253Eh1%2520heading%2520dreamUI%253C%252Fh1%253E%2520%2520%2520%2520%2520%2520%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ch2%253Eh2%2520heading%2520dreamUI%253C%252Fh2%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ch3%253Eh3%2520heading%2520dreamUI%253C%252Fh3%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ch4%253Eh4%2520heading%2520dreamUI%253C%252Fh4%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ch5%253Eh5%2520heading%2520dreamUI%253C%252Fh5%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ch6%253Eh6%2520heading%2520dreamUI%253C%252Fh6%253E"
+  sandbox="allow-scripts allow-same-origin">
```
Comment (aryan-13) [inline documents/documents.html]: A code-formatter can be used to beautify the code!

**Review by kaustubh-jsr — APPROVED** [review]

**Review by aryan-13 — COMMENTED** [review]
The class names and the code is clean! Indentation can be improved!

File: documents/documents.html (hunk 2 above)
Comment (dikshadhote) [reply documents/documents.html]: Formatted

**Review by aryan-13 — APPROVED** [review]

**Pull request merged**